EIPS/eip-7620.md

```diff
@@ -289,34 +163,24 @@ These rules were not adopted to avoid the extra logic and complexity (esp. the l
 
 This change poses no risk to backwards compatibility, as it is introduced at the same time EIP-3540 is. The new instructions are not introduced for legacy bytecode (code which is not EOF formatted), and the contract creation options do not change for legacy bytecode.
     
-Legacy create transactions with `data` starting with `EF00` now are invalid. Similarly `CREATE` and `CREATE2` calls with `EF00` initcode fail early without executing the initcode. Previously, in both cases the initcode execution would begin and fail on the first undefined instruction `EF`.
+`CREATE` and `CREATE2` calls with `EF00` initcode fail early without executing the initcode. Previously, in both cases the initcode execution would begin and fail on the first undefined instruction `EF`.
 
 ## Test Cases
 
-Creation transaction, `CREATE` and `CREATE2` cannot have its *code* starting with `0xEF`, but such cases are covered already in [EIP-3541](./eip-3541.md). However, new cases must be added where creation transaction, `CREATE` or `CREATE2` have its *initcode* being (validly or invalidly) EOF formatted:
+Creation transaction, `CREATE` and `CREATE2` cannot have its *code* starting with `0xEF`, but such cases are covered already in [EIP-3541](./eip-3541.md). However, new cases must be added where `CREATE` or `CREATE2` have its *initcode* being (validly or invalidly) EOF formatted:
```

Comment:
```suggestion
 `CREATE` and `CREATE2` cannot have its *code* starting with `0xEF`, but such cases are covered already in [EIP-3541](./eip-3541.md). However, new cases must be added where `CREATE` or `CREATE2` have its *initcode* being (validly or invalidly) EOF formatted:
```

Reply:
Why? eip3541 still applies to creation txs, if they execute legacy initcode and it returns `0xEF`

Reply:
ah right I confused it with initcode, sorry